aptos-node/src/network.rs

```diff
@@ -263,6 +268,14 @@ pub fn setup_networks_and_get_interfaces(
     )
 }
 
+/// Creates a bounded executor for the given network config and runtime
+fn create_bounded_executor(network_config: &NetworkConfig, runtime: &Runtime) -> BoundedExecutor {
```

Comment:
nit: `create_bounded_executor` name is pretty generic. A better naming might be `create_deserialization_executor` ?

Reply:
This should be no longer needed now that we removed the bounded executor requirement 😄